.github/workflows/ruff.yml

```
@@ -5,4 +5,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: astral-sh/ruff-action@v1
+      - uses: astral-sh/ruff-action@v2
+        with:
+          args: "format --check"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)